### ISSUE TITLE
"chunk section" -> "section"

### ISSUE THREAD

**Issue by Runemoro** [asiekierka/MinecraftPOMF#1142]
Right now we have a mix of both "chunk section" and just "section" to refer to 16x16x16 regions. I think we should shorten this to "section" everywhere, since "section" is a common enough term to refer to them. A few vanilla strings also refer to them as just "sections".

**Comment by Earthcomputer** [comment]
I see it as the phrase "chunk section" which means 16x16x16 regions, and "section" on its own as meaningless, and too vague, if you don't already happen to know what it means. "section" is arguably fine in parameter names where "chunk" can be inferred from other context.

**Comment by liach** [comment]
Now that mojang has tweaked height and sections may become the basic unit of loading (this is already the case for entities). Will we reconsider naming this `section`?

**Comment by liach** [comment]
After thinking, I agree that we should treat this thing as "chunk section". We do have "item stack" and frequently call it `stack`, yet "stack" on its own does not mean an item stack. Similar logic should go here.

This has been open for more than 1 year. The vote as of this comment is 6&times;:-1:. If there is no extra comments or votes in 24 hours, the current vote will be taken as the result.

**Comment by Earthcomputer** [comment]
If we are to rename it, I'd rename it to subchunk, which is widely used in the niche tech community and I think is a better name than chunk section, but chunk section is the more widely accepted name for it.

**Comment by liach** [comment]
Seeing that mojang is already using them a lot since the expansion of height. I guess they may have a colloquial name for it one day in the release notes, and we can go and see by that point. (though mojang colloquial names aren't always very good either, like "chat component" is just... you know)

**Comment by liach** [comment]
So the vote is clear. We won't name this section until mojang does some big mechanism changes, in case we shall revisit again.
